css/main.css

```diff
@@ -4,7 +4,155 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+/* Hello TA,s i had some Problems to find a nice solution to get the box around login button and the home button, i guess i did some mistakes on the way. Looking forward for solution */
+
+
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* 1 point */
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  
+  background: linear-gradient(to bottom, rgba(28, 83, 128, 0.8) 30%, #fff 30%  );
+}
+
+/* 1point */
+h1 {
+  color: #06365f;
+}
+
+.content {
+  
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  
+  margin-top: 100px;
+  margin-bottom: 100px;
+  height: 60vh;
+}
+
+.main-content {
+  width: 60vh;
+    
+}
+
+
+.main-content  {
+  display: flex;
+  flex-direction: column;
+  
+}
+
+.phones {
+  width: 30vh;
+  display: flex;
+  padding: 20px;
+  margin-right: 50px;
+}
+
+.logo-home  {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.instagram-logo {
+  display: flex;
+}
+
+.login-menu {
+  display:flex;
+  flex-direction: row;
+  align-items: center;
+  
```

Comment:
To solve your problem of box:   
  background-color: #5E85A6;
  padding: 5px;
  border: 4px solid #4C7597;